lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -113,14 +113,17 @@ public Thread StartTaskOnNewThread()
                 }
                 catch (TaskStartHandlerException e)
                 {
+                    Logger.Log(Level.Info, "TaskRuntime::TaskStartHandlerException");
```

Comment:
I don't think we need to make it Warning-level, it doesn't warn us about the exception itself but just keeps track of where it was thrown (the exception itself is logged later).
Once this goes to production, we'll need to go through all our code and check which logs should be which level.